lighthouse-cli/index.js

```diff
@@ -61,6 +61,9 @@ const cli = yargs
   ], 'Configuration:')
   .describe({
     'mobile': 'Emulates a Nexus 5X',
```

Comment:
let's nuke the mobile flag. sorry i wasn't explicit about that but that's what i meant to say in #646